cli/azd/cmd/testdata/TestUsage-azd-provision.snap

```diff
@@ -10,10 +10,11 @@ Usage
   azd provision [flags]
 
 Flags
-        --docs               	: Opens the documentation for azd provision in your web browser.
-    -e, --environment string 	: The name of the environment to use.
-    -h, --help               	: Gets help for provision.
-        --preview            	: Preview changes to Azure resources.
+        --disable-deployment-state 	: Do not use latest Deployment State (bicep only).
```

Comment:
Suggest naming it -ignore or --force or --skip-cache or ignore-cache instead of -disable since it is a one off operation. Disable to me means it will be disabled in future calls as well.

I would also like a little more info in the help statement.    To state that it won't look at local cache.

Reply:
updated to --ignore-deployment-state